app/components/campusDisplay.js

```diff
@@ -0,0 +1,27 @@
+import React, {Component} from 'react'
+import { connect } from 'react-redux';
+import StudentItem from './studentItem'
+
+class CampusDisplay extends Component {
+  constructor(props) {
+    super(props)
+    console.log('initializing CampusDisplay')
```

Comment:
Also be mindful of cleaning up the work surface `console.log`s, etc. as you get through your debugging.